### ISSUE TITLE
Segfault

### ISSUE THREAD

**Issue by ayoungethan** [stsdc/monitor#220]
On Hera 5.1.7

Dmesg:
```
com.github.stsd[15351]: segfault at 8 ip 00007fcff1c94618 sp 00007fcfd902e730 error 4 in libpixman-1.so.0.34.0[7fcff1c3a000+9d000]
Code: 10 48 c1 e2 04 e8 08 f5 fa ff 48 8b 44 24 28 48 8b 40 10 4c 01 60 08 48 8b 7c 24 38 e8 e1 f3 fa ff 48 8b 44 24 28 48 8b 78 10 <48> 8b 47 08 85 c0 0f 85 54 07 00 00 48 83 3f 00 74 05 e8 c1 f3 fa
```
Beautiful and simple program, thank you!

**Comment by stsdc** [comment]
@ayoungethan Does it happen with a newest release?

**Comment by ayoungethan** [comment]
Will check. "Show Indicator" appears to no longer work (no indicator appears under wingpanel when option is enabled).

**Comment by ayoungethan** [comment]
Yes, it seems much more stable now! I will post back/reopen if more crashes are noticed.

Thank you!

**Comment by stsdc** [comment]
Released 0.9.4. Indicator issue should be fixed. Remember to restart wingpanel or reboot.
